src/main/java/io/cdap/plugin/gcp/bigtable/source/HBaseColumn.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.gcp.bigtable.source;
+
+/**
+ * This class represents a reference to HBase column
+ */
+public class HBaseColumn {
+  private static final String FAMILY_QUALIFIER_DELIMITER = "__";
+  private final String family;
+  private final String qualifier;
+
+  private HBaseColumn(String family, String qualifier) {
+    this.family = family;
+    this.qualifier = qualifier;
+  }
+
+  public static HBaseColumn fromFullName(String fullName) {
+    int delimiterIndex = fullName.indexOf(FAMILY_QUALIFIER_DELIMITER);
```

Comment:
I think isValidName() is always called before this, but it is safer to check that this is not negative and throw an exception with a descriptive message if it is invalid. This makes it impossible for the caller to use this class incorrectly and it also isolates the logic about what is a valid field name to just this class.

Reply:
done